src/main/webapp/js/utils/inputs/date-picker.jsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ReactDatePicker from 'react-datepicker';
 import 'react-datepicker/dist/react-datepicker.css';
-import { parseISO } from 'date-fns';
+import { parseISO, format } from 'date-fns';
 
 const DatePicker = ({
-  onChange, value, dateFormat, ...props
+  onChange, value, dateFormat, showTimeSelect, ...props
 }) => {
   const handleChange = (date) => {
-    onChange(date);
+    const val = showTimeSelect ? date.toISOString() : format(date, 'YYYY-MM-DD');
```

Comment:
change it to:
`if (!date || typeof date === 'string') {`
`  onChange(date);`
`} else {`
`  onChange(showTimeSelect ? date.toISOString() : format(date, 'yyyy-MM-dd'));`
`}`
because if the date will be null or string it will fail in this line when `date.toISOString()` is called